src/nvmesh.js

```diff
@@ -1114,8 +1119,9 @@ export class NVMesh {
     } else {
       obj = NVMeshLoaders.readFreeSurfer(buffer)
     } // freesurfer hail mary
-    console.log('mesh loaded')
-    console.log('obj', obj)
+    if (Object.hasOwn(obj, 'anatomicalStructurePrimary')) {
```

Comment:
This could probably be
```js
if (obj.anatomicalStructurePrimary) {
  anatomicalStructurePrimary = obj.anatomicalStructurePrimary
}
```

or even

```js
const anatomicalStructurePrimary = obj.anatomicalStructurePrimary || ''
```

Reply:
@jens-ox I have implemented your former approach. While your latter approach wins for brevity, as an old C programmer it lacks clarity to my eye.